### ISSUE TITLE
Translation(s) that need work - no output compared to Xliff-sync

### ISSUE THREAD

**Issue by fvet** [rvanbekkum/ps-xliff-sync#8]
@rvanbekkum 

I've a project with following XliffSync settings:

```
"xliffSync.baseFile": "Navitrans 365 Core.g.xlf",
    "xliffSync.findBySource": true,
    "xliffSync.missingTranslation": "%EMPTY%",
    "xliffSync.autoCheckNeedWorkTranslations": true,
    "xliffSync.autoCheckMissingTranslations": true,
```

When manually synching g.xliff to nl-be.xliff (via `synchronize translation units` in VSCode), I get a message for 2 missing translations (ok) and **1 translation that needs work** (fine !!)

![image](https://user-images.githubusercontent.com/25268332/114011642-c211d700-9865-11eb-99bc-b6ec76d77efc.png)

This is indeed marked in the nl-be.xliff file

`<target state="needs-adaptation">Opmerkingen</target>`

Re-running the  `synchronize translation units` in VSCode, gives the same message: 2 missing translations (ok) and **1 translation that needs work** (fine !!)

----

I've integrated the above in a pipeline using PS-xliff-sync, to ensure translations are synced as part of the build process and we use the output as build artifact, to have an idea on the amount of work needed to translate our apps. This works for the missing translations, but seems not to list the 'translations that need work'. 

Running a PS script to check on missing translations and to check for problems ...

```
$WorkspaceFolder = 'C:\GIT\Navitrans.Core\' 
$SourceFile = Join-Path $WorkspaceFolder "App\Translations\Navitrans 365 Core.g.xlf"
Sync-XliffTranslations -sourcePath $SourceFile -targetPath (Join-Path $WorkspaceFolder "App\Translations\Navitrans 365 Core.nl-BE.xlf") -reportProgress
Check-XliffTranslations -targetPath (Join-Path $WorkspaceFolder "App\Translations\Navitrans 365 Core.nl-BE.xlf") -checkForMissing -reportProgress -checkForProblems -translationRules ConsecutiveSpacesConsistent, ConsecutiveSpacesExist, OptionLeadingSpaces, OptionMemberCount, Placeholders
```

... only provide this output

```
Detected: 2 missing translation(s).
Missing translation in unit 'Enum 870401921 - EnumValue 3355985101 - Property 2879900210'.
Missing translation in unit 'Enum 870401921 - EnumValue 4008471815 - Property 2879900210'.
Detected: 0 translation(s) that need work.  <<<< ?

```

I expected to see the same result as in VSCode, that 1 translation needed work (instead of 0).

Is there something wrong with my settings?

[Navitrans 365 Core.zip](https://github.com/rvanbekkum/ps-xliff-sync/files/6277901/Navitrans.365.Core.zip)

**Comment by rvanbekkum** [comment]
Hi @fvet,
I just checked and I see the issue, vsc-xliff-sync has the following statement which will include trans-units with state needs-adaptation in the count that is missing in ps-xliff-sync:
```ts
    if (targetDocument.getState(unit) === translationState.needsWorkTranslation) {
        return true;
    }
```
This should be an easy thing to fix. I will include it with a new release!

**Comment by fvet** [comment]
Hi @rvanbekkum , any idea on when we might expect the new release? ;)

**Comment by rvanbekkum** [comment]
Hi @fvet,
I will probably work on it this weekend!

**Comment by fvet** [comment]
@rvanbekkum You should enjoy the nice weather and the re-opening of the bars instead this weekend :)

**Comment by rvanbekkum** [comment]
A change for this has been shipped in version 1.2.0.0.
